### PR TITLE
Add meme minigun

### PR DESCRIPTION
**Pull request by StrawWagen**

### CODE REVIEW

File: lua/cfc_entity_stubber/m9k/heavy_weapons/m9k_minigun.lua [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
```
Comment (plally) [inline lua/cfc_entity_stubber/m9k/heavy_weapons/m9k_minigun.lua:1]: I noticed the m9k ak example did AddCSLuaFile. You might need to do that here as well so the changes are mirrored on the client.

Comment (StrawWagen) [reply lua/cfc_entity_stubber/m9k/heavy_weapons/m9k_minigun.lua:1]: oops

File: lua/cfc_entity_stubber/m9k/heavy_weapons/m9k_minigun.lua [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
```
Comment (plally) [inline lua/cfc_entity_stubber/m9k/heavy_weapons/m9k_minigun.lua:1]:
```suggestion
```

File: lua/cfc_entity_stubber/m9k/heavy_weapons/m9k_minigun.lua [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+AddCSLuaFile()
+
+CFC_M9k_Stubber.registerStub( function()
+    local SWEP = weapons.GetStored( "m9k_minigun" )
```
Comment (wrefgtzweve) [inline lua/cfc_entity_stubber/m9k/heavy_weapons/m9k_minigun.lua:5]:
```suggestion
    local SWEP = cfcEntityStubber.getWeapon( "m9k_ak47" )
```
Use the dedicated stubber function instead

Comment (wrefgtzweve) [reply lua/cfc_entity_stubber/m9k/heavy_weapons/m9k_minigun.lua:5]: https://github.com/CFC-Servers/cfc_entity_stubber/commit/e42e28b115ac944e7fd1e27bb6936935fef602f5
these are stub examples you can take a look at

File: lua/cfc_entity_stubber/m9k/heavy_weapons/m9k_minigun.lua [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    SWEP.Primary.KickUp = 1.5
+    SWEP.Primary.KickDown = 0.2
+    SWEP.Primary.KickHorizontal = 0.2
+    SWEP.Primary.Automatic = true
+    SWEP.Primary.Ammo = "ar2"
+    SWEP.Primary.NumShots = 2
+    SWEP.Primary.Damage = 6
+    SWEP.Primary.Spread = .06
+    SWEP.Primary.IronAccuracy = .08
+
```
Comment (plally) [inline lua/cfc_entity_stubber/m9k/heavy_weapons/m9k_minigun.lua:20]:
```suggestion
```

**Pull request merged**